vdev/src/commands/package/msi.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+
+#[cfg(windows)]
+use {
+    crate::{app, util},
+    std::env,
+    std::fs,
+    std::iter::once,
+    std::path::Path,
+};
+
+/// Create a .msi package for Windows
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        #[cfg(windows)]
+        {
+            // TODO: wait for other PR to be merged then replace the below line with app::version()
+            let archive_version = env::var("VERSION").or_else(|_| util::read_version())?;
+
+            // rm -rf target/msi-x64
+            // cp -R distribution/msi target/msi-x64
```

Comment:
Better commentary here would at least point at why you are doing this, not what the original shell commands were, particularly since the functions are named so clearly in this instance. For example:
```suggestion
            // Make sure we start with a fresh `msi-x64` target directory.
```

Reply:
I will work on better commentary in the next commit